### PR TITLE
OSS-FIPS

### PR DESCRIPTION
**Pull request by bmckercher123**
### Add FIPS mode doc to OSS

- For OSS 3.25/next

### CODE REVIEW

File: calico/operations/fips.mdx [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+When running in FIPS compliant mode, {{prodname}} uses FIPS-approved cryptographic algorithms and NIST-validated cryptographic modules.
+
+## Concepts
+
+The Federal Information Processing Standards (FIPS) are publicly announced standards developed by the National Institute of Standards and Technology for use in computer systems by government agencies and government contractors. {{prodname}} FIPS mode is enabled during installation by:
+
+- Switching the cryptographic modules for the golang-based applications to use the FIPS-140-2 validated [Tigera Cryptographic Module](https://csrc.nist.gov/projects/cryptographic-module-validation-program/certificate/4095)
+- Configuring TLS servers and other cryptographic functions to use FIPS 140-2 approved cryptographic algorithms
+- Leveraging third-party cryptographic modules for other applications:
```
Comment (rene-dekker) [inline calico/operations/fips.mdx]: You can remove line 20+21, those are not used in OSS.

Comment (lwr20) [reply calico/operations/fips.mdx]: Also fluentd not used in OSS

Comment (rene-dekker) [reply calico/operations/fips.mdx]: I meant remove:
```
- Leveraging third-party cryptographic modules for other applications:
- Fluentd is built using [Red Hat Enterprise Linux 8 OpenSSL Cryptographic Module](https://csrc.nist.gov/projects/cryptographic-module-validation-program/certificate/3842)
```

Comment (bmckercher123) [reply calico/operations/fips.mdx]: Ah, right! Removed.

**Review by rene-dekker — APPROVED** [review]
lgtm

**Pull request merged**